game/scripts/vscripts/components/creeps/creep_power.lua

```diff
@@ -10,7 +10,7 @@ function CreepPower:GetPowerForMinute (minute)
     return {   0,        1.0,      1.0,      1.0,      1.0,      1.0,      1.0 * self.numPlayersXPFactor}
   end
 
-  if minute > 60 then
+  if (minute > 40 and PointsManage:GetLimit() = 50) or (minute > 50 and PointsManage:GetLimit() = 100) or (minute > 120 and PointsManage:GetLimit() = 200) then
```

Comment:
It also should be `PointsManager` I believe. It might also be better to use a lookup table that maps kill limit to time of onset.
i.e. something like
```local exponentialGrowthTime = {
  50 = 40,
  100 = 50,
  200 = 60
}
if minute > exponentialGrowthTime(PointsManager:GetLimit()) then
...```